### PR TITLE
created create item, update item. Moved destroy method to make tests …

### PR DESCRIPTION
**Pull request by nvnatasha**
In this PR:

- route and method for create item done
- route and method for update item done
- moved destroy method in items controller.

### CODE REVIEW

**Review by wally-yawn — CHANGES_REQUESTED** [review]
You'll need some sad path tests for both create and update, I think you can mimic the sad path tests James did in merchant_create_spec and merchant_update_spec

**Comment by wally-yawn** [comment]
Looking at it again, looks like the items model will also need to validate the presence of those attributes and would need model tests for those, Looks like the merchant model and tests have a pattern you can follow

**Review by wally-yawn — APPROVED** [review]
Approve tests and error handling, new card made for model tests

File: app/controllers/api/v1/items_controller.rb [hunk 1]
```diff
@@ -14,21 +14,56 @@ def show
...
+
+    render json: ItemSerializer.format_single_item(item)
+  rescue ActiveRecord::RecordNotFound => error
+    render json: error_messages([error.message], 404), status: 404
+  rescue ActiveRecord::RecordInvalid => errors
+    render json: error_messages(errors.record.errors.full_messages, 422), status: 422
+  end
+  end
+
+  def destroy
```
Comment (Jonathan-Atkins) [inline app/controllers/api/v1/items_controller.rb:40]: thanks for moving that

**Review by Jonathan-Atkins — APPROVED** [review]

**Pull request merged**